src/lib.rs

```diff
@@ -88,8 +87,8 @@ impl Default for DspPlugin {
 
 impl Plugin for DspPlugin {
     fn build(&self, app: &mut App) {
-        app.insert_resource(DspManager::new(self.sample_rate))
-            .add_asset::<DspSource>();
+        app.init_resource::<DspManager>()
```

Comment:
Why is `init_resource` used instead of `insert_resource`?

Reply:
Thanks for catching this ❤️.  I thought `insert_resource` was deprecated, but only `add_asset` was. This also works with `insert_resource` so I will switch it back.